api/client/src/main/java/org/projectnessie/client/auth/oauth2/Secret.java

```diff
@@ -38,27 +31,11 @@ public int length() {
     return value.length;
   }
 
-  public char[] getCharsAndClear() {
```

Comment:
This is *in theory* part of the public API, but I don't think anybody out there was calling this method or `getStringAndClear`. But I'm happy to put them back and mark them as deprecated instead.